bin/write-mappings.js

```diff
@@ -14,15 +14,15 @@ const mappingNames = [
   'by_recap_customer_code',
   'by_sierra_location',
   'by_statuses',
-  'by_catalog_item_type'
+  'by_catalog_item_type',
+  'by_fulfillment'
 ]
 
 const filesWritten = []
 
```

Comment:
Does this have to be an await Promise.all(mappingNames.map(async ...etc situation? not sure if this forEach will be awaited otherwise.

Reply:
Oh I think I see. This is just to update the local mappings for tests, correct?

Reply:
Yes correct. The `forEach` will not block, and that's maybe a little reckless, but okay because nothing happens after the `forEach` block.